test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertFrom-Json.Tests.ps1

```diff
@@ -1,6 +1,32 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-Describe 'ConvertFrom-Json' -tags "CI" {
+
+function New-NestedJson {
+    Param([int] $Depth)
+
+    $nestedJson = "true"
+
+    $Depth..1 | ForEach-Object {
+        $nestedJson = '{"' + $_ + '":' + $nestedJson + '}'
+    }
+
+    return $nestedJson
+}
+
+function Count-ObjectDepth {
+    Param([PSCustomObject] $InputObject)
+
+    for ($i=1; $i -le 100000; $i++)
+    {
+        $InputObject = Select-Object -InputObject:$InputObject -ExpandProperty:$i
```

Comment:
Convention is not to have the colon for parameters (except for switches if need to explicitly set to opposite)

```suggestion
        $InputObject = Select-Object -InputObject $InputObject -ExpandProperty $i
```